dev/travis/before_script.sh

```diff
@@ -80,8 +80,8 @@ case $TEST_SUITE in
         curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
         export NVM_DIR="$HOME/.nvm"
         [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
-        bash -c "nvm install $NODE_JS_VERSION" || true
```

Comment:
That's absolutely fine as I had just an approximate understanding of what is going on when reviewed your code as well 👍 

Hope we have some decent DevOps in Magento Community or in Magento Team to review Travis scripts :)